README.md

```diff
@@ -753,7 +753,9 @@ Cobra can generate documentation based on subcommands, flags, etc. Read more abo
 
 ## Generating shell completions
 
-Cobra can generate a shell-completion file for the following shells: Bash, Zsh, Fish, Powershell. If you add more information to your commands, these completions can be amazingly powerful and flexible.  Read more about it in [Shell Completions](shell_completions.md).
+Cobra can generate a shell-completion file for the following shells: Bash, Zsh, Fish, Powershell. Custom, dynamic completions are supported too. If you add more information to your commands, these completions can be amazingly powerful and flexible.  Read more about it in [Shell Completions](shell_completions.md).
+
+*Note: For generating **custom** shell completions for `zsh`, [use the master branch](https://stackoverflow.com/a/56410071/9606036) in your `.mod` file.*
```

Comment:
The intention of helping people with this extra note is good.  However, it would mean that the maintainers would have to remember to remove this line before doing a release.

Personally, I think people have to understand that the documentation on the master branch matches the code on the master branch.  We shouldn't start documenting old code in newer versions.

As an alternative could there be a way to put a link at the top of the readme to say: "For the documentation of the latest release, please see here (link to last release tag)". We'd need a link that wouldn't need to be updated if possible.